Doc/library/pathlib.rst

```diff
@@ -1510,9 +1510,11 @@ call fails (for example because the path doesn't exist).
 
 .. method:: Path.symlink_to(target, target_is_directory=False)
 
-   Make this path a symbolic link to *target*.  Under Windows,
-   *target_is_directory* must be true (default ``False``) if the link's target
-   is a directory.  Under POSIX, *target_is_directory*'s value is ignored.
+   Make this path a symbolic link to *target*.  Under Windows, if the target
+   is present, the type of the symlink will be created to match.  Otherwise,
+   the symlink will be created as a directory if *target_is_directory* is
+   ``True`` or a file symlink (the default) otherwise.  Under POSIX,
+   *target_is_directory*'s value is ignored.
```

Comment:
```suggestion
   Make this path a symbolic link to *target*.
   Under Windows, if the target is not present, you must specify whether the symlink
   is a file (default) or directory (by setting *target_is_directory* to ``True``).
   If the target exists, this will be inferred automatically. 
   Under POSIX, *target_is_directory*'s value is ignored.
```

Reply:
Overall it's clearer, but "must" makes it sound like passing *target_is_directory* is necessary if the target doesn't exist.

Reply:
We could also copy the entire wording from `os.path.symlink()` if it's useful: https://docs.python.org/3.13/library/os.html#os.symlink

Reply:
> We could also copy the entire wording from `os.path.symlink()` if it's useful: https://docs.python.org/3.13/library/os.html#os.symlink

Having refreshed my memory of that wording, I think it is much clearer - I'd support using it here.